worker/lib/workers/qemu.ts

```diff
@@ -188,6 +205,12 @@ class QemuWorker extends EventEmitter implements Leviathan.Worker {
 		});
 	}
 
+	private async iptablesRules(bridgeName: string, bridgeAddress: string){
+		await execProm(`iptables-legacy -I FORWARD -i ${bridgeName} -j ACCEPT -m comment --comment ${this.iptablesComment}`);
+		await execProm(`iptables-legacy -I FORWARD -o ${bridgeName} -m state --state RELATED,ESTABLISHED -j ACCEPT -m comment --comment ${this.iptablesComment}`);
+		await execProm(`iptables-legacy -t nat -A POSTROUTING ! -o ${bridgeName} --source ${bridgeAddress}/24 -j MASQUERADE -m comment --comment ${this.iptablesComment}`);
```

Comment:
These rules could probably be simplified. The first two should be unnecessary, and the last rule could be simplified to `iptables-legacy -t nat -A POSTROUTING -j MASQUERADE -m comment ...`

We can optionally specify the output device, which should be the external interface, but this is not necessary.

Reply:
made the change and tested this locally - works well thanks :)

Reply:
@jakogut , I removed the first 2 rules, but for the last rule I had to specify the interface, otherwise my laptop couldn't connect to the internet while the rule was active.